egnoramoose.py

```diff
@@ -1,119 +1,167 @@
 class Pawn:
-	def __init__(self,location):
-		self.location = location
+    pass
+
+
+class Location:
+    def __init__(self, *location):
+        self.row = -1
+        self.column = -1
+        self._convert(location)
+
+    def __sub__(self, other):
+        return Location(self.row-other.row, self.column-other.column)
+
+    def __str__(self):
+        return '{}{}'.format(self.row, self.column)
+
+    @staticmethod
+    def _list_to_int(list_obj):
```

Comment:
Turns everything in a list to an integer if it can. This is my helper to change a string to integers, a list of integers to integers, or two separate single char strings to 2 separate integers